components/LocalFooter/LocalFooterMvp.tsx

```diff
@@ -46,14 +46,15 @@ export const LocalFooterMvp = () => (
           <Heading as="h3" size={3} leading="tight">
             We can do this, together
           </Heading>
-          <Paragraph variant="big" leading="display">
+          <Paragraph variant="big" leading="cozy">
             With your support, this vision will become reality.
           </Paragraph>
           <CtaLink
             href={links.ood.areasToSupport}
             variant="ghost-swipe"
             color="white"
-            icon="arrow-right"
+            icon="external"
+            animate="top-right"
```

Comment:
Alexis requested for this "Find ways to give" link to have the external arrow to match the main nav